src/models/logic/cccChange.ts

```diff
@@ -183,6 +183,27 @@ export async function stakeDeposit(
     );
 }
 
+export async function reportDoubleVote(
+    params: {
+        address: string;
+        change: U64;
+        blockNumber: number;
+        transactionHash: string;
+    },
+    options: {
+        transaction?: Transaction;
+    } = {}
+): Promise<CCCChangeInstance> {
+    return createCCCChange(
+        {
+            ...params,
+            reason: "report",
+            isNegative: false
+        },
+        options
+    );
```

Comment:
shouldn't we create a negative change for the criminal?

Reply:
No. The criminal's deposit is already decreased when it sent self-nominations.
Only the informant's balance is increased.

Reply:
I see.